### PR TITLE
Access floor from floors module

### PR DESCRIPTION
**Pull request by colbertk**
## Type of change
- [x ] Feature

## Description of change
Using `getFloor` on bid request, if available, to retrieve the floor value.

### CODE REVIEW

File: modules/tripleliftBidAdapter.js [hunk 1]
```diff
@@ -138,6 +138,22 @@ function _buildPostBody(bidRequests) {
...
+  let floor = null;
+  if (typeof bid.getFloor === 'function') {
+    let floorInfo = bid.getFloor({
+      currency: 'USD',
+      mediaType: 'banner',
+      size: _sizes(bid.sizes)
+    });
+    if (typeof floorInfo === 'object' &&
+    floorInfo.currency === 'USD' && !isNaN(parseInt(floorInfo.floor))) {
+      floor = parseFloat(floorInfo.floor);
```
Comment (willchapin) [inline modules/tripleliftBidAdapter.js:151]: is there a reason to use `parseFloat` here but `parseInt` above?

File: modules/tripleliftBidAdapter.js [hunk 2]
```diff
@@ -138,6 +138,22 @@ function _buildPostBody(bidRequests) {
...
+    }
+  }
+  return floor || bid.params.floor;
```
Comment (willchapin) [inline modules/tripleliftBidAdapter.js]: would `0` ever be a valid floor?

Comment (colbertk) [reply modules/tripleliftBidAdapter.js]: good catch!

**Review by kzhouTL — APPROVED** [review]

File: modules/tripleliftBidAdapter.js [hunk 3]
```diff
@@ -138,6 +138,22 @@ function _buildPostBody(bidRequests) {
   return data;
 }
 
+function _getFloor (bid) {
+  let floor = null;
+  if (typeof bid.getFloor === 'function') {
+    let floorInfo = bid.getFloor({
```
Comment (willchapin) [inline modules/tripleliftBidAdapter.js]: this should be `const` i think

**Review by willchapin — APPROVED** [review]
👍 👍

**Pull request merged**